2024.12/fpm-alpine/Dockerfile

```diff
@@ -129,6 +129,8 @@ RUN set -ex; \
         echo 'post_max_size=${PHP_UPLOAD_LIMIT}'; \
     } > /usr/local/etc/php/conf.d/friendica.ini; \
     ln -s /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini; \
+    \
```

Comment:
Please remove the generated files from the PR.

Reply:
Files deleted from PR